source/elements/tags/Alert/README.md

```diff
@@ -0,0 +1,10 @@
+# Alert
+
```

Comment:
Some documentation here would be great! Even if it's minimal! I think some explanation on the contexts of each variant would be very useful here.

Reply:
I have updated documentation and extended "global colors" within our variables. In addition, I have extended the alert type based on level it's being used, as it seems to be a common trend across projects (page/form and field-level type)

That was also partially the reason why I had that body inner container for alerts that may be page-level where the colored background extends to edges of your viewport but the messaging retains body copy width and is centered. 

Additionally, I assumed we may have an example with close button as well.

Reply:
I think this conversation is important. I think what we want to add to FC are elements that are atomic enough we don't need to generalize them. There are good thoughts on including that internal container, and a close button. But because those are things we have to 'cut' to make this new tag general enough for FC, is that a component we want to include in our starting library? Thoughts @drolsen @elseloop @bstaruk ?

Reply:
With @bstaruk we’ve been debating hat maybe some of these tags don’t necesaeily need to merged into master, maybe just keep them in dev, or have a branch with our internal components we tend to reuse a lot on projects. 

When it comes to alerts, I’ve been seeing them as one of the common tags in our pattern libraries, as almost every site out there has some form of notifications. 

Let’s definiteky keep talking on what we should consider adding to our base and what we should archive some place for easy reuse upon needs.

Reply:
I'm seeing this as a "when needed" vs. baseline tag as well. However, I don't want us to lose these (and more) great pieces of work by not capturing them somewhere useful for us. I think it might be time for us to create a third branch called "library" that sits along develop and master. 

I propose each dev will need to determine if what they are pushing is going to be for this new library branch or baseline develop from now on and PR accordingly. 

We will continue or normal review process to ensure we all have a chance to agree upon any changes requested. As we start new projects, we simply keep library in our back pockets and pick what we need, when we need, and modify from there.

Thoughts?

Reply:
@drolsen I have pushed few more updates and improvements to this tag, including your svg fix https://github.com/connectivedx/fuzzy-chainsaw/pull/314.